File: scala/src/main/scala/org/agilegrenoble/objectcalisthenics/Item.scala
package org.agilegrenoble.objectcalisthenics

import scala.reflect.{BeanProperty, BooleanBeanProperty}
//remove if not needed
import scala.collection.JavaConversions._

class Item(@BeanProperty var name: String, @BeanProperty var sellIn: Int, @BeanProperty var quality: Int)
   

File: c#/GuildedRose/Item.cs
using System;

public class Item {
	private String name;
	private int sellIn;
	private int quality;

	public Item(string name, int sellIn, int quality) {
		this.name = name;
		this.quality = quality;
		this.sellIn = sellIn;
	}

    public int SellIn
    {
        get { return sellIn; }
        set { sellIn = value; }
    }

    public int Quality
    {
        get { return quality; }
        set { quality = value; }
    }
    
    public string Name
    {
        get { return name; }
    }
}
